projects/layout/components/cell/cell.styles.less

```diff
@@ -3,6 +3,7 @@
 [tuiCell] {
     --tui-height: calc(var(--tui-height-s) - 0.5rem);
```

Comment:
Is this used anywhere?

Reply:
No, it used `--t-height` only
It's misspel and maybe rename `--tui-height` to `--t-height` on host OR just remove `--tui-height` ?

<img width="607" alt="image" src="https://github.com/user-attachments/assets/c5d68f02-a874-42f6-a85c-13f8099c10c0">
<img width="698" alt="image" src="https://github.com/user-attachments/assets/018b92f2-53f3-4413-82b3-e1e22b3a4316">